docs/controls/wpf-winforms/WebViewCompatible.md

```diff
@@ -37,37 +36,33 @@ Unless specified otherwise in this article, the documentation for the [WebViewCo
 
 ## Feature limitations
 
-When compared to the UWP **WebView** control, the current release of the WPF and Windows Forms **WebView** controls has some limitations. For the complete list of these limitations, see [Known Issues of the WebView control for Windows Forms and WPF applications](WebView-known-issues.md).  The WebViewCompatible control has further limitations because it exposes a common interface that is a subset of common functionality across the two underlying implementations.  In a system that supports the Windows 10 WebViewControl, this control will exhibit those limitations plus further exclusions to the exposed functionality. In a legacy environment, this control will use WebBrowser, and have the same additional interface limitations.
-
-## Add the WebViewCompatible control to the Visual Studio Toolbox
-
-First, open the Visual Studio **Toolbox**, then right-click anywhere in the toolbox, and select the **Choose Items** option.
+When compared to the UWP **WebView** control, the current release of the WPF and Windows Forms **WebView** controls has some limitations. For the complete list of these limitations, see [Known Issues of the WebView control for Windows Forms and WPF applications](WebView-known-issues.md). The **WebViewCompatible** control has further limitations because it exposes a common interface that is a subset of common functionality across the two underlying implementations. In a system that supports the Windows 10 **WebViewControl**, this control will exhibit those limitations plus further exclusions to the exposed functionality. In a legacy environment, this control will use **WebBrowser**, and have the same additional interface limitations.
 
-<!-- Remove these steps before going public. This is for early adopters only -->
+## Add the WebViewCompatible control to the Visual Studio Toolbox for Windows Forms applications
 
-### For Windows Forms applications
+1. Install the [Microsoft.Toolkit.Forms.UI.Controls.WebView](https://www.nuget.org/packages/Microsoft.Toolkit.Forms.UI.Controls.WebView) NuGet package.
 
-1. In the **.NET Framework Components** tab of the **Choose Toolbox Items** dialog box.
+2. Open the Visual Studio **Toolbox**, right-click anywhere in the toolbox, and select the **Choose Items** option.
```

Comment:
No longer required in Visual Studio 15.8

Reply:
Thanks for the feedback. I split up the Windows Forms steps for VS 15.8 (just install the package and the control shows up in the toolbox) and earlier versions of VS (install the package and manually add the assembly to the toolbox).